src/index.js

```diff
@@ -1,52 +1,125 @@
 'use strict';
 
+const fs = require('fs');
+const FreeSurfer = require('freesurfer-parser');
 const pkg = require('../package.json');
 
+/**
+ * Get FreeSurfer region of interest picker.
+ *
+ * @param {string} feature
+ * @returns {Function} FreeSurfer data picker
+ */
+const getFreeSurferDataPicker = feature => freeSurferData => {
+  return new FreeSurfer({ string: freeSurferData })[feature];
+};
+
+/**
+ * Read file.
+ *
+ * @param {string} file
+ * @returns {Promise} Resolves to `file`'s contents
+ */
+const readFileAsync = file => new Promise((resolve, reject) => {
+  fs.readFile(file, (error, data) => {
+    if (error) {
+      reject(error);
+    } else {
+      resolve(data.toString());
+    }
+  });
+});
+
+/**
+ * Get normalized tags.
+ *
+ * @param {Object} file
+ * @param {Object} file.tags Hash map of tag names to values
+ * @returns {Array} Normalized tags
+ */
+const getNormalizedTags = file => Object.keys(file.tags).sort().reduce(
+  (memo, tag) => {
+    const value = file.tags[tag];
+
+    if (typeof value === 'boolean') {
+      return memo.concat(value === true ? 1 : -1);
+    } else if (typeof value === 'number') {
+      return memo.concat(value);
+    }
+
+    return memo;
+  },
+  []
+);
+
 module.exports = {
   name: pkg.name,
   version: pkg.version,
-  local: {
+  local: [{
```

Comment:
Local now contains two computations: the first pre-processes data piped in from COINSTAC, and the second passes the data to _ridge_regress.py_.